docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.6'
+
+volumes:
+  db-vol:
+
+services:
+  db:
+    image: mysql
+    command: --default-authentication-plugin=caching_sha2_password
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: photo_portfolio
+      MYSQL_USER: web_user
+      MYSQL_PASSWORD: password
+
+    volumes:
+      - db-vol:/var/lib/mysql
+
+    ports:
+      - "3306:3306"
+
+  db_seeder:
+    image: mysql
+    volumes:
+      - ./mysql-seed.sql:/mysql-seed.sql
```

Comment:
Is the docker file going to only be used for development? I suggest that an `init.sql` file is also used so that some data is available immediately for the development of features. This way we don't really need to persist the data while in development which is a good rule in general. You don't want to keep test data on your machine all the time.

Reply:
Yeah, I did intend to do that in a separate MR, but I'm also fine with adding it here. I'll spend a little time to make up some test data.

Reply:
Ok, I added some dummy data in the seed script - of course we can fill this out with better data once we get more into the details.

Reply:
I did start the docker with this data in place and then I could connect to the db and see that the data was there.